.github/pull_request_template.md

```diff
@@ -14,17 +13,20 @@ Fixes # (issue)
 - [ ] Documentation (updates to documentation or READMEs)
 - [ ] Chore (any other change that doesn't affect source or test files)
 
-# Properties of change
+**Properties**
 
 - [ ] Breaking change (this change will cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 
-# Test plan
+Release Notes: if the change is end-user visible, also consider providing some notes for the release engineer.
```

Comment:
Separate **Release Notes** section instead after **Test plan**?

Reply:
Sure, but I do think "does it need documentation" and "is it a breaking change" are related to the release notes documentation

Reply:
That's a good point. Maybe **Properties** should just be **Release Notes**?

Reply:
Tried another formulation, how do you like it?